<launch>
<!-- Launch file for map stitch -->

	<arg name="rviz" default="true" />

    <rosparam file="$(find backend_optimization)/config/map_stitch.yaml" command="load" />

    <node pkg="backend_optimization" type="map_stitch" name="system" output="screen" /> 

	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find backend_optimization)/rviz_cfg/map_stitch.rviz" />
	</group>

</launch>